### DISCUSSION TITLE
Problem installing SharpPcap NuGet package in VS2015

### DISCUSSION THREAD

**Discussion by Terry550** [chmorgan/sharppcap#264] [General]
Hello,
Looking for help. I have VS2015 and DotNetFramework V4.6.1 for a new C# Project.
When trying to install the NuGet package I get the following Error:

Error		Could not install package 'SharpPcap 5.4.0'. You are trying to install this package into a project that targets '.NETFramework,Version=v4.6.1', but the package does not contain any assembly references or content files that are compatible with that framework. For more information, contact the package author.		

I've tried different frameworks but get the same error message.
Any suggestions would be greatly appreciated.
Thanks
Terry

**Comment by kayoub5** [comment]
SharpPcap targets .NET Standard 2.0, You will need NuGet client 3.6 or higher installed.
See https://docs.microsoft.com/en-us/dotnet/standard/net-standard#net-standard-libraries-and-visual-studio
> If you only need to consume .NET Standard 2.0 libraries in your projects, you can also do that in Visual Studio 2015. However, you need NuGet client 3.6 or higher installed. You can download the NuGet client for Visual Studio 2015 from the NuGet downloads page.

**Reply by Terry550** [reply]
Thank! I will give that a shot and post back to the discussion.

**Comment by Terry550** [comment]
Installing NuGet client 3.6 worked for installing SharpPcap 5.4.0 thanks, but now I'm getting 29 duplicate reference errors when I try to build (see below). I tried editing my project file but still keep getting the errors. Do I need to uninstall something?
Thanks for any help you can provide.
Terry

Severity	Code	Description	Project	File	Line	Suppression State
Error	CS1703	Multiple assemblies with equivalent identity have been imported: 'C:\Program Files (x86)\MSBuild\Microsoft\Microsoft.NET.Build.Extensions\net461\ref\System.Xml.XDocument.dll' and 'C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.6.1\Facades\System.Xml.XDocument.dll'. Remove one of the duplicate references.	EMS_EthernetMonitorServer_WPCAP_NPCAP	C:\Users\M228220\Documents\Visual Studio 2015\Projects\EMS_EthernetMonitorServer_WPCAP_NPCAP\CSC	1	Active

**Reply by kayoub5** [reply]
Without seeing the csproj file, there is no way to tell what's going on here.
That aside, do you REALLY need to work with Visual Studio 2015?

**Reply by Terry550** [reply]
Finally figured it out, Thanks kayoub5 for your help!!!
I found the answer on this github link
https://github.com/dotnet/sdk/issues/1539

Answer: It looks like MSBuild is still adding the facades from the targeting pack. I haven't chased down the diff yet, but you should be able to workaround by setting ImplicitlyExpandDesignTimeFacades=false.
